README.md

```diff
@@ -1,111 +1,121 @@
+# TexTron Snake 🐍
 
- # TexTron Snake
- 
- TexTron Snake is a very simple ASCII snake game tributed to the classic
- Tron arcade. It was meant to be developed as a collaborative programming
- exercise in a course on Open Source Systems taught to undergraduate CS
- students.
+TexTron Snake is a very simple ASCII snake game tributed to the classic
+Tron arcade. It was meant to be developed as a collaborative programming
+exercise in a course on Open Source Systems taught to undergraduate CS
+students.
 
- ## INSTALL
+## SETUP
 
- If you have obtained the project source from the __version control repository__,
+In the following section, the leading `#` in commands indicates that it requires
+administrative privileges (`sudo`).
 
- execute the script 
+### Dependencies
+
+
+If you have obtained the project source from the __version control
+repository__, you'll need to have GNU Build System (Autotools) installed.
+In Debian/Ubuntu based platforms, you may install the required software with:
 
  ```
- $ ./autogen.sh
+ # apt install automake autoconf
  ```
 
-to boostrap the build configuration script `configure`. To that end, you'll 
-need to have GNU Build System (Autotools) installed. In debian/ubuntu based 
-platforms,  you may install required software with
+Other missing dependencies will be indicated by the configuration script ---
+e.g.: if you don't have `libncurses` installed, it'll complain about it and you
+may install it (if in Debian/Ubuntu based system) with:
 
 ```
-$ sudo apt install automake autoconf
+# apt install libncurses5-dev
 ```
```

Comment:
lines 21 and 29 should have '$' and not '#' right?

Was unable to create a proper suggestion. Github breaks if I try to add code blocks inside suggestions 😭

Reply:
Don't think so, as you need `sudo` to run those commands

Reply:
Oh my bad. You right :P